src/unix/bsd/apple/b32/mod.rs

```diff
@@ -70,7 +75,7 @@ cfg_if! {
         impl Eq for pthread_attr_t {}
         impl ::fmt::Debug for pthread_attr_t {
             fn fmt(&self, f: &mut ::fmt::Formatter) -> ::fmt::Result {
-                f.debug_struct("pthread_attr_t")
+                f.debug_struct("pthread_once_t")
```

Comment:
I think this might not have been intentional

Reply:
Fixed--thanks for catching this!